### ISSUE TITLE
Gradle sync doesn't complete with version 6.0.8

### ISSUE THREAD

**Issue by zsmb13** [mikepenz/MaterialDrawer#2312]
## About this issue

Trying to build the sample app on the `develop` branch, changed the direct module dependency

```groovy
implementation project(':library')
```

to the Gradle dependency specified in the readme:

```groovy
implementation("com.mikepenz:materialdrawer:6.0.8@aar") {
    transitive = true
}
```

The Gradle sync fails with the following errors:

```
Failed to resolve: com.mikepenz:library-core:3.2.6
Failed to resolve: com.mikepenz:library:3.2.6
```

I also had this issue in my own project, I just thought it's easier to reproduce with the official sample app.

---

Here's what I found so far about the source of the issue by looking at the dependency trees:

```
+--- com.mikepenz:materialdrawer:6.0.8
|    |    +--- com.mikepenz:fastadapter:3.2.6
|    |    |    +--- com.android.support:appcompat-v7:27.1.1 (*)
|    |    |    +--- com.android.support:recyclerview-v7:27.1.1 (*)
|    |    |    \--- com.android.support:support-annotations:27.1.1
|    |    \--- com.mikepenz:fastadapter-extensions-expandable:3.2.6
|    |         +--- com.mikepenz:library-core:3.2.6 FAILED
|    |         +--- com.mikepenz:library:3.2.6 FAILED
```

**Comment by mikepenz** [comment]
Thanks for reporting you might need to include the sub dependencies manually for the time being (disable transitive) as I have difficulties with jcenter (there seems to be a mixup) I'll let you know when these problems are resolved

**Comment by thomasdsouza** [comment]
@mikepenz , Thank you for a superbly flexible libarary!

Can you please give a bit more info of your suggested workaround above (how to include sub dependencies manually)?
I tried the following which didn't work...

    implementation 'com.mikepenz:materialdrawer:6.0.8@aar'
    implementation 'com.mikepenz:fastadapter:3.2.6'
    implementation 'com.mikepenz:fastadapter-extensions-expandable:3.2.6'
    implementation 'com.mikepenz:library-core:3.2.6'
    implementation 'com.mikepenz:library:3.2.6'

**Comment by mikepenz** [comment]
newest release now available

**Comment by zsmb13** [comment]
Oh, got it, it works after cleaning Gradle caches! Thanks!

**Comment by thomasdsouza** [comment]
Yes fix st good and works as expected.
@mikepenz, you are the best... Have a wonderful future!

**Comment by Ogafais** [comment]
ERROR: Failed to resolve: com.mikepenz:materialdrawer:6.0.2
ERROR: Failed to resolve: com.mikepenz:materialdrawer:6.0.1

i got this error can you help me

**Comment by sbm08** [comment]
> ERROR: Failed to resolve: com.mikepenz:materialdrawer:6.0.2
> ERROR: Failed to resolve: com.mikepenz:materialdrawer:6.0.1
> 
> i got this error can you help me

have you solved the problem?

**Comment by pratham3012** [comment]
Failed to resolve: com.mikepenz:aboutlibraries:6.0.1
Failed to resolve: com.mikepenz:materialdrawer:6.0.2

Please help me solve these two errors.

**Comment by mikepenz** [comment]
@pratham3012 please use the latest 6.x version released v6.0.8 or so

also no support anymore for 6.x releases. you should upgrade to v8
